features/shopcarts.feature

```diff
@@ -7,3 +7,15 @@ Scenario: The server is running
     When I visit the "home page"
     Then I should see "Shopcart Demo REST API Service"
     Then I should not see "404 Not Found"
+
+Scenario: List all shopcarts
+    When I visit "shopcarts"
+    Then I should see "Settlers of Catan"
```

Comment:
you will use I should see product1 and I should NOT see product2
in the list of shopcarts, you should see all products.